src/pages/Register/RegisterForm.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import { connect } from 'react-redux';
+//styles
+import classNames from 'classnames/bind';
+import styles from './RegisterPage.css'
+import 'coupon-components/build/styles.css';
+//components
+import { InputBox, Button, Typography, Card } from 'coupon-components';
+import { FormattedMessage } from 'react-intl';
+import { Link } from 'react-router-dom';
+import Logo from 'Components/Logo/Logo';
+
+const cx = classNames.bind(styles);
+
+class RegisterForm extends React.Component {
+  state = {
+    credentials: {
+      name: '',
+      company: '',
+      email: '',
+      password: ''
+    }
+  }
+
+  onChange = (event) => {
+    this.setState({
```

Comment:
Si ya estamos usando redux form ya come todas las variables que tenga en el onSubmit(values) no es  necesario este this.setState.